### ISSUE TITLE
Some questions about your paper/code

### ISSUE THREAD

**Issue by wvangansbeke** [wrlife/RNN_depth_pose#3]
Hi Rui,
I have a few questions regarding your paper. 

1. Do have the RMSE number when you train the same network without LSTMs and not on small sequences? In this way, the influence of the LSTMs can be clearly evaluated. I couldn't find this number in the paper.

2. Why are you reading from the file with test sequences when you are making the dataloader for training?

3. I noticed that the reported RMSE values for the test set are much better than SOTA. However, delta1 and delta2 don't show a lot of improvement. Could you share your predicted depth files for the test set or can you clarify these results?

4. Do you have any idea why your unsupervised depth map in the first row of figure 4 doesn't look good? E.g., at the boundaries of the car. I would expect much better results with those numbers.

Kind regards,
Wouter
